_posts/2022-12-01-athena.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title:  "Trino delivers for Amazon Athena"
+author: Manfred Moser, Martin Traverso
+image: /assets/trino-light.png
+excerpt_separator: <!--more-->
+---
+
+It is time to reach out to another large group of Trino users. The release of
```

Comment:
I feel like this intro paragraph needs a slightly better hook or a little more excitement.

Reply:
I hope the new update is better